lib/views/quiz/quiz_result.dart

```diff
@@ -60,13 +60,13 @@ class QuizResult extends StatelessWidget {
                         style: Theme.of(context)
                             .textTheme
                             .body1
-                            .copyWith(fontSize: 100),
+                            .copyWith(fontSize: 100, color: Colors.white),
```

Comment:
This shouldn't use the body1 theme as we no longer use it also the text themes that are used have a default color white so thats redundant

Reply:
Are you able to update for me?

Reply:
will do

Reply:
changes made